mjcf_to_sdformat/mjcf_to_sdformat/converters/world.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (C) 2022 Open Source Robotics Foundation
+#
+# Licensed under the Apache License, version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from mjcf_to_sdformat.converters.link import mjcf_geom_to_sdf
+
+import sdformat as sdf
+
+
+def mjcf_worldbody_to_sdf(mjcf_model, world):
+    """
+    Convert a MJCF worldbody to a SDFormat world
+
+    :param mjcf.Element mjcf_model: The MJCF model
```

Comment:
This should be `mjcf.RootElement` since we are accessing `worldbody` below. 
Consider also renaming `mjcf_model` to `mjcf_root`.

Reply:
https://github.com/gazebosim/gz-mujoco/pull/26/commits/40148462713bdbba5fa3ba236d57bf8c24596e61